### PR TITLE
feat: add biomed to collaborations

### PR DESCRIPTION
**Pull request by tdivoll**
This PR adds the Biomed diversity survey collaboration to the list of collaborations!

### CODE REVIEW

File: schemas/collaborations.yml [hunk 1]
```diff
@@ -29,7 +29,8 @@ id: "enum(
           'aer-group',
           'wilhelmus-lab',
           'the-policy-lab',
-	  'acadia'
+	        'acadia',
```
Comment (hollandjg) [inline schemas/collaborations.yml:32]:
```suggestion
          'acadia',
```

Comment (hollandjg) [reply schemas/collaborations.yml:32]: looks like there's an out of place tab here

**Review by hollandjg — CHANGES_REQUESTED** [review]
just one minor issue – it's that tab again.

**Review by eldu — APPROVED** [review]

**Review by hollandjg — APPROVED** [review]
🚀 Sorry! I Ignored the re-review notification here!

**Pull request merged**